examples/basic_functions.py

```diff
@@ -7,6 +7,7 @@
 config = dict(email=email, pwd=pwd, name=name)
 
 client = CornFlow(url="http://127.0.0.1:5000")
```

Comment:
I would not have this url appear her, although it has appeared in the client repository so we should ask if David can change it